src/bot/commands/helpers/checkElevatedRights.ts

```diff
@@ -0,0 +1,9 @@
+import Config from '@config/Config';
+import { Collection, Role } from 'discord.js';
+
+const userHasElevatedRole = (rolesCache: Collection<string, Role>) => {
+    const config = new Config();
+    return rolesCache.some(r => config.rolesAllowedToRunCommands.includes(r.name));
+}
+
+export default userHasElevatedRole;
```

Comment:
Let's rename the file of this helper to be the same as the function within it.

Reply:
Renamed to `userHasElevatedRole`